### PR TITLE
Isolating Sources of Disentanglement in VAEs

### PR DESCRIPTION
**Pull request by nathanpainchaud**
Warning that this review is quite math-heavy, so the option to view the file using GitHub markdown rendering is pretty broken.

### CODE REVIEW

File: article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+although the modifications proposed by both papers differ.
+
+Lets recall the objective of the $$\beta$$-VAE:
+
+$$
+\tag{1} \mathcal{L}_{\beta} = \frac{1}{N} \sum_{n=1}^{N} (\mathbb{E}_{q} [\log p(x_{n}|z)] - \beta KL(q(z|x_{n})~\|~p(z)))
+$$
+
+where $$\beta \gt 1$$ to encourage disentanglement. This is a more general formulation of the VAE, since we can obtain
+the original VAE objective by setting $$\beta = 1$$ in eq. 1.
```
Comment (jhlegarreta) [inline article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md:36]: _eq. 1_ -> Equations are not numbered when introducing them using LaTeX, right? At least not the way we usually write them; I ignore if the LaTeX `equation` environment can be used.

File: article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+The authors describe the purpose of each term:
+1. The *index-code mutual information* (MI) is the mutual information $$I_{q}(z;n)$$ between the data variable and
+latent variable;
+2. The *total correlation* (TC) is a generalization of mutual information to more than two random variables. A heavier
+penalty on this term is what encourages a disentangled representation;
+3. The *dimension-wise* KL prevents individual latent dimensions from deviating too far from their priors.
+
+## Training with Minibatch-Weighted Sampling
+The decomposed expression from eq. 2 requires to evaluate the aggregate posterior, which depends on the entire dataset
```
Comment (jhlegarreta) [inline article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md:62]: _eq. 2_ -> We do not have equation numbering when writing equations with LaTeX, right?

File: article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md (hunk 1 above)
Comment (nathanpainchaud) [reply article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md:36]: Good question! That's one of KaTeX's functionalities that I discovered. You can number your equation by adding `tag{eq_tag}` before it (inside the `$$` markers), where `eq_tag` is the tag you want your equation to display.

[Here is an example](https://katex.org/docs/supported.html#annotation) from KaTeX's documentation.

File: article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md (hunk 2 above)
Comment (nathanpainchaud) [reply article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md:62]: See my answer above.

File: article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md (hunk 1 above)
Comment (jhlegarreta) [reply article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md:36]: :+1: Sorry, my bad. I did not pay attention to the `tag{}` that was inside the math environment. Thanks for the explanation.

File: article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md (hunk 2 above)
Comment (jhlegarreta) [reply article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md:62]: :+1:, sorry again, and thanks for the explanation.

File: article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md (hunk 1 above)
Comment (nathanpainchaud) [reply article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md:36]: No need to be sorry. I only just learned about it recently myself :wink:

**Pull request merged**